test/extended/testdata/test-deployment-test.yaml

```diff
@@ -14,7 +14,10 @@ spec:
         execNewPod:
           containerName: myapp
           command:
-          - /bin/echo
+          - /bin/sh
+          - -c
+          args:
+          - echo
```

Comment:
@kargakis there is one using this, I was trying to avoid adding more tests/fixtures to speed up the tests. This change will cause to use both cmd and args so if one of them breaks we will get a failure.